spirv_glsl.hpp

```diff
@@ -602,6 +602,7 @@ class CompilerGLSL : public Compiler
 		bool allow_precision_qualifiers = false;
 		bool can_swizzle_scalar = false;
 		bool force_gl_in_out_block = false;
+		bool force_meged_mesh_block_hlsl = false;
```

Comment:
Typo? A backend flag should not mention a specific language.

Reply:
At first I was thinking that this option makes sense only for hlsl (actually metal would need it as well). 
Replaced to `force_meged_mesh_block`